pancancer_evaluation/utilities/tcga_utilities.py

```diff
@@ -82,7 +82,7 @@ def process_y_matrix(
         suffixes=("_count", "_proportion"),
     ).merge(filter_disease_df, left_index=True, right_index=True)
 
-    if not test:
+    if (not test) and (output_directory is not None):
```

Comment:
Technically you don't need parens here because of [logical operator precedence](https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html), but they are useful for grouping so inbd

Reply:
Yeah, I like to add them for logical tests like this because I personally find it a bit clearer (the parens visually set apart the two things I'm trying to compare), but good to know that it isn't strictly necessary.